lib/schema/schema.graphql

```diff
@@ -374,23 +371,26 @@ type User {
   Users can enter a short description to be prominently displayed along with their name and other info.
   """
   oneLiner: String
+  user: User!
   countryOfResidence: Country
+  gender: Gender
+  educationLevel: EducationLevel
+  channels: [Channel!]
+  channelInvitations: [ChannelInvitation!]
+  channelParticipants: [ChannelParticipant!]
+  companies: [Company!]
```

Comment:
The Company array still seems to be supplied here - am I missing something?

Reply:
The query in question returns a User, not a UserExt